### DISCUSSION TITLE
Is it possible to convert back and forth between JWT and CWT?

### DISCUSSION THREAD

**Discussion by Evertt** [panva/jose#274] [Q&A]
CWT being the binary compressed version of JWT. I'm reading a lot about it, but I haven't yet found any npm packages that are as extensive as this jose package. So it would be nice if I could just use this jose package most of the time and only convert the JWT to a CWT at the end of the process.

Is that possible? If it isn't yet, would you be willing to write a function for that? 👼

**Comment by panva** [comment]
> Is that possible?

It is not.

> If it isn't yet, would you be willing to write a function for that? 👼

I would not.

CBOR would need to be a package of its own similar to this one, but there's no way I would weave them together.

**Reply by Evertt** [reply]
Ah, that's too bad for me then. I have found a few cose packages yesterday but they're all made for NodeJS. Oh well, maybe I'll just stick with JWT for now then.
